content/projects/infrastructure/index.adoc

```diff
@@ -107,7 +107,7 @@ Here is a non-exhaustive list of services that we provide and maintain.
 | http://archives.jenkins.io[Archives]       | https://github.com/jenkins-infra/helpdesk/labels/archives[GitHub issues]       | https://github.com/jenkins-infra/jenkins-infra/blob/production/dist/profile/manifests/archives.pp[Puppet Code]
 | https://ci.jenkins.io[Core and Plugin CI]  | https://github.com/jenkins-infra/helpdesk/labels/ci.jenkins.io[GitHub issues]  | https://github.com/jenkins-infra/documentation/blob/main/ci.adoc[User Docs], https://github.com/jenkins-infra/jenkins-infra[Deployment scripts]
 | https://get.jenkins.io/[Mirrors]           | https://github.com/jenkins-infra/helpdesk/labels/get.jenkins.io[GitHub issues] | link:https://github.com/jenkins-infra/helm-charts/tree/main/charts/mirrorbits[Mirrorbits helm chart], link:https://get.jenkins.io/war-stable/latest/jenkins.war.sha256?mirrorlist[mirror list], link:https://get.jenkins.io/war-stable/latest/jenkins.war?mirrorstats[mirror stats], link:/download/mirrors/[Running a mirror]
-| http://mirrors.jenkins.io/[Old Mirrors]    | `mirrors.jenkins.io`      | link:https://github.com/jenkins-infra/jenkins-infra[old mirrors.jenkins.io Puppet scripts] (search for `mirrorbrain`), link:https://github.com/jenkins-infra/infra-mirror[old mirror scripts], link:http://mirrors.jenkins-ci.org/status.html[old status page]
+| http://mirrors.jenkins.io/[Old Mirrors]    | `mirrors.jenkins.io`      | link:https://github.com/jenkins-infra/jenkins-infra[old mirrors.jenkins.io Puppet scripts] (search for `mirrorbrain`), link:https://github.com/jenkins-infra/infra-mirror[old mirror scripts]
```

Comment:
This should also be removed, since it has been obsolete for a long time.  Thanks for detecting it.  Links to archived repositories aren't a lot of help when we are trying to diagnose a current infrastructure issue.  I'd rather remove this line that retain it as links to something that we no longer use
```suggestion
```